runltp-ng

```diff
@@ -64,7 +65,8 @@ GetOptions(
 	"setup" => \$setup,
 	"cmd=s" => \$cmd,
 	"verbose" => \$verbose,
-	"timeout=i" => \$timeout
+	"timeout=i" => \$timeout,
+	"openqa_format" => \$openqa_formatter
```

Comment:
I guess that we can make this more generic in a case there will be need to add another output formatter.

So what about parameter "json_filter" => \$json_filter and if this parameter is defined we would call results::$json_filter on the results before passing them to the writelog function?